bindings/nodejs/lib/types/block/address.ts

```diff
@@ -118,12 +133,105 @@ class NftAddress extends Address {
     }
 }
 
+/**
+ * An implicit account creation address.
+ */
+class ImplicitAccountCreationAddress extends Address {
+    /**
+     * The Ed25519 address.
+     */
+    readonly address: Ed25519Address;
+    /**
+     * @param address An Ed25519 address.
+     */
+    constructor(address: Ed25519Address) {
+        super(AddressType.ImplicitAccountCreation);
+        this.address = address;
+    }
+
+    toString(): string {
+        return this.address.toString();
+    }
+}
+
+const RestrictedAddressDiscriminator = {
+    property: 'type',
+    subTypes: [
+        { value: Ed25519Address, name: AddressType.Ed25519 as any },
+        { value: AccountAddress, name: AddressType.Account as any },
+        { value: NftAddress, name: AddressType.Nft as any },
+        {
+            value: ImplicitAccountCreationAddress,
```

Comment:
Restricted can't be ImplicitAccountCreation

Reply:
oh ok